src/containers/StructurePage/utils.ts

```diff
@@ -6,4 +6,27 @@
  *
  */
 
+import { MatomoPageData, MatomoResponse } from "../../modules/matomo/matomoApi";
+
 export const capitalizeFirstLetter = (str: string) => str.charAt(0).toUpperCase() + str.slice(1);
+
+export interface ResourceStats extends Pick<MatomoPageData, "nb_visits" | "sum_time_spent"> {
+  year: string;
+}
+
+export const transformMatomoData = (data: PromiseSettledResult<MatomoResponse>[]): Record<string, ResourceStats> => {
+  const transformed = data.reduce<Record<string, ResourceStats>>((acc, cur) => {
+    if (cur.status === "fulfilled") {
+      const year = Object.keys(cur.value)[0];
+      if (!year || !cur.value[year].length) return acc;
+      const { pathname } = new URL(cur.value[year][0].url);
+      acc[pathname] = {
+        year: year,
+        nb_visits: cur.value[year][0].nb_visits,
+        sum_time_spent: cur.value[year][0].sum_time_spent,
```

Comment:
Tenker dei er mest intressert i avg_time_on_page for å sjå kor lenge sida vises i snitt.

Reply:
Trello kan sikkert være lurt, men bestilling er:
verdiar det er verdt å ta med:
* "nb_visits": 1312,
* "nb_hits": 2012,
* "avg_time_on_page": 94,
På sikt
* "bounce_rate": "32 %",
* "exit_rate": "50 %",